DataProcessing/datax-host/src/main/scala/datax/fs/HadoopClient.scala

```diff
@@ -124,21 +125,24 @@ object HadoopClient {
     * @param vaultName key vault name to get the key of storage account
     * @param sa name of the storage account
     */
-  private def resolveStorageAccount(vaultName: String, sa: String) = {
+  def resolveStorageAccount(vaultName: String, sa: String) : String = {
```

Comment:
If a function can return NULL, then best practice is for the function to return Option[String] instead of String

For examples of this pattern, see
DataProcessing/datax-utility/src/main/scala/datax/utility/DataGenerator.scala
/DataProcessing/datax-host/src/main/scala/datax/securedsetting/KeyVaultClient.scala

Reply:
done